### ISSUE TITLE
"Comments" link on the user profile should include changeset and note comments.

### ISSUE THREAD

**Issue by DeeHants** [openstreetmap/openstreetmap-website#1148]
The user profile page has a navbar link titled "(My )comments".
While the link goes to `/user/Deanna%20Earley/diary/comments`, it's not obvious from the text that it's just diary notes.
I think it would be far more worthwhile to include changeset and note comments in the list.

I accept that "(My )notes" shows the initial text of notes I've commented on (but not the actual comment), and #842 discusses listing changeset comments.

**Comment by tomhughes** [comment]
I agree that the name of the link is misleading, but I'm not convinced the proposed solution is the correct one. My initial thought is simple that the title should be improved and, if required, new links added to view other sorts of comments.

I believe there are already issues asking for links to other sorts of comments, so in some senses this is a duplicate although I don't think anybody has proposed a unified view before. A unified view would certainly be a much larger project.

**Comment by tomhughes** [comment]
One related issue is #842.

**Comment by DeeHants** [comment]
I was going to suggest a rename, but felt that that detracted from the main request.

**Comment by Dimitar5555** [comment]
This issue, along with #842 should be marked as resolved by #5518.
